sentry_sdk/integrations/_wsgi_common.py

```diff
@@ -51,6 +51,12 @@ def request_body_within_bounds(client, content_length):
 
 
 class RequestExtractor:
+    """
+    Base class for request extraction. It does not make sense to make this an ABC because it is not used
+    for typing, only so that child classes can inherit common methods from it. Only some child classes
+    implement all methods that raise NotImplementedError in this class.
+    """
+
```

Comment:
I'd remove this (or leave it in as a comment rather than a docstring) -- it's more of a note for us than for users.

Reply:
I left the first sentence as a docstring, since it lets users know the purpose of the class, and I split the rest into a normal comment.